frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/validations/providers/v2vvmware-tab-validation.ts

```diff
@@ -18,14 +18,25 @@ export const getV2VVMwareImportProvidersTabValidity = (
 
   const hasSelectedVM = !!iGetVMWareFieldValue(state, id, VMWareProviderField.VM);
   const hasLoadedVM = !!iGetVMWareFieldAttribute(state, id, VMWareProviderField.VM, 'vm');
+  // check if host name has any kind of whitespace
+  const isHostnameValid = !/\s/.test(iGetVMWareFieldValue(state, id, VMWareProviderField.HOSTNAME));
```

Comment:
the BZ is just about spaces at the end, am I missing something ?

Reply:
why not check for any spaces while we're here? :)

Reply:
can we truncate the hostname ?